components/tests/ui/resources/robot.template

```diff
@@ -20,7 +20,7 @@ ${SERVER_ID}            1
 ${BROWSER}              Firefox
 ${DELAY}                0
 ${WAIT}                 50
-${TIMEOUT}              1
+${TIMEOUT}              60
```

Comment:
Just to be clear, this means the timeout of all tests - not just Delete tests - is extended to 1 minute, correct?